src/status_im2/setup/datetime.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im2.setup.datetime
```

Comment:
@flexsurfer should I also `require` this namespace into a file like src/status_im2/setup/core.cljs? so that it gets registered?

Reply:
I also need to include it somewhere for the tests too.. any ideas where is best? 🤔